src/data/admin/Committees.tsx

```diff
@@ -2,8 +2,37 @@ import { generateAffiliation, generateSelect, generateStatus } from "./Columns";
 import { AFFILIATIONS } from "../form/Information";
 import { ICONS } from "./Icons";
 import { STATUSES } from "@/data/Statuses";
+import { ColumnDef } from "@tanstack/react-table";
+import { ReactNode } from "react";
 
-export const TAGS = [
+type TAGS = {
```

Comment:
this is getting extracted out, I would talk to Daniel and Taaha and finding a standard to follow especially regarding some other aspects of the dashboard as well